liger_iris_pipeline/tests/test_image2.py

```diff
@@ -42,25 +62,38 @@ def test_image2_subarray(tmp_path):
     input_model.meta.subarray.ystart = ystart + 1
     input_model.meta.subarray.xsize = xsize
     input_model.meta.subarray.ysize = ysize
+
+    # Subarray indices
     subarray_slice = np.s_[ystart:ystart+ysize, xstart:xstart+xsize]
 
+    # Slice the data
+    # NOTE: .err must be first since it's not in the FITS file and is created on the fly when accessed
+    input_model.err = np.array(input_model.err[subarray_slice])
     input_model.data = np.array(input_model.data[subarray_slice])
     input_model.dq = np.array(input_model.dq[subarray_slice])
-    input_model.err = np.array(input_model.err[subarray_slice])
 
-    raw_science_subarray_filename = tmp_path / "temp_subarray_science.fits"
+    # Save the subarray science frame
+    raw_science_subarray_filename = "temp_subarray_science.fits"
```

Comment:
why have you removed `tmp_path`? it should be better to have a temp location so files get wiped after tests run

Reply:
I think this is addressed below with paths in `test_asn.json`. `tmp_path` was originally an input arg to the subarray test function.

Reply:
Fixed below in [e50837c](https://github.com/oirlab/liger_iris_pipeline/pull/75/commits/e50837c99efa6c7a98d61b1efecb27e06d130a98), thanks for pointing this feature out!